src/scripts/core/services/score-maths.ts

```diff
@@ -15,12 +15,15 @@ export class ScoreMaths {
     ]);
 
     static getScore(skin: number): number | undefined {
-        if(this.scoreMap.has(skin)) {return this.scoreMap.get(skin)}
-        else return undefined;
+        if(this.scoreMap.has(skin)) {
+            return this.scoreMap.get(skin);
+        }
+        else 
+            return undefined;
```

Comment:
You have `{}` around the if but none around the else, I don't mind which way you decide to do it but both should be the same